dataset/dictionary/enemies.json5

```diff
@@ -959,52 +1119,69 @@
   {
     en: "Rifthound Whelp",
     ja: "獣域ウェルプ",
+    zh: "兽境幼兽",
     pronunciationJa: "じゅういきウェルプ",
+    variants: {
+      zh: [ "小流血狗" ],
+    },
     notes: "whelp はイヌ科の動物の子供を指す語。",
     tags: [ "enemy" ],
   },
   {
     en: "Rockfond Rifthound Whelp",
     ja: "獣域ウェルプ・岩",
+    zh: "兽境幼兽·岩",
     pronunciationJa: "じゅういきウェルプ・いわ",
     notes: "whelp はイヌ科の動物の子供を指す語。",
     tags: [ "enemy" ],
   },
   {
     en: "Thundercraven Rifthound Whelp",
     ja: "獣域ウェルプ・雷",
+    zh: "兽境幼兽·雷",
     pronunciationJa: "じゅういきウェルプ・かみなり",
     notes: "whelp は若いイヌ科の動物を指す語。",
     tags: [ "enemy" ],
   },
   {
     en: "Rifthound",
     ja: "獣域ハウンド",
+    zh: "兽境猎犬",
     pronunciationJa: "じゅういきハウンド",
+    variants: {
+      zh: [ "流血狗", "大流血狗" ],
+    },
     tags: [ "enemy" ],
   },
   {
     en: "Rockfond Rifthound",
     ja: "獣域ハウンド・岩",
+    zh: "兽境猎犬·岩",
     pronunciationJa: "じゅういきハウンド・いわ",
     tags: [ "enemy" ],
   },
   {
     en: "Thundercraven Rifthound",
     ja: "獣域ハウンド・雷",
+    zh: "兽境猎犬·雷",
     pronunciationJa: "じゅういきハウンド・かみなり",
     tags: [ "enemy" ],
   },
   {
     en: "Golden Wolflord",
     ja: "黄金王獣",
+    zh: "黄金王兽",
     pronunciationJa: "おうごんおうじゅう",
     tags: [ "enemy", "inazuma" ],
   },
   {
     en: "Wolves of the Rift",
     ja: "獣域ウルブズ",
+    zh: "兽境猎犬",
```

Comment:
This seems wrong translation.

![Wolves of the Rift (zh)](https://user-images.githubusercontent.com/107226297/174476047-57569844-e1ad-4831-ab6d-7e011779792b.png)
![Wolves of the Rift (en)](https://user-images.githubusercontent.com/107226297/174476021-ceaf155f-8195-42e1-bf24-4d4fa5e7ee9b.png)

Reply:
Sorry for my carelessness. This may be the same error of the Snezhnayan Maiden and the left may be caused by my carelessness and the misleading of Japanese expression. All have been fixed.